.github/CODEOWNERS

```diff
@@ -0,0 +1,41 @@
+# This file list the owners of the sub-part of our monorepo.
+# We assign those parts to a specific team whose members are proficient in those elements.
+# If you think you belongs to one of those teams because you're proficient in one of those parts, ask @touilleMan.
```

Comment:
```suggestion
# This file lists the owners of the sub-parts in our monorepo.
# These parts are assigned to a specific team whose members are proficient in them.
# If you think you belongs to one of these teams because you're proficient in one of those parts, ask @touilleMan.
```